tests/Functional/ContentRenderTestBase.php

```diff
@@ -87,20 +95,28 @@ protected function getStorage(string $entity_type_id): EntityStorageInterface {
    *   Rendered element.
    * @param string $name
    *   Name of the document.
+   * @param string $language
+   *   Document language.
+   * @param string $meta
+   *   Size and format of the file.
+   * @param string $link
+   *   Link to the file.
+   * @param string $button_label
+   *   Text on the button.
    */
-  protected function assertMediaDocumentDefaultRender(NodeElement $element, string $name): void {
+  protected function assertMediaDocumentDefaultRender(NodeElement $element, string $name, string $language, string $meta, string $link, string $button_label): void {
```

Comment:
The button label is hardcoded in the media template, we should remove it from the assertion method, as it's something that cannot be changed by the application, so we don't need to test it. Let's only test the button URL.

![image](https://user-images.githubusercontent.com/153362/116368378-905ab300-a808-11eb-8f71-82d837f41d26.png)

Reply:
"Download" is translatable string and it can be different on different languages.
Also in some another template it can be changed

Reply:
Ok, let's leave it.